sdk/documentintelligence/azure-ai-documentintelligence/README.md

```diff
@@ -250,6 +253,19 @@ for page in result.pages:
                 f"'{selection_mark.polygon}' and has a confidence of {selection_mark.confidence}"
             )
 
+if result.paragraphs:
+    print(f"----Detected #{len(result.paragraphs)} paragraphs in the document----")
+    for paragraph in result.paragraphs:
+        print(f"Found paragraph with role: '{paragraph.role}' within {paragraph.bounding_regions} bounding region")
+        print(f"...with content: '{paragraph.content}'")
+
+    result.paragraphs.sort(key=lambda p: (p.spans.sort(key=lambda s: s.offset), p.spans[0].offset))
```

Comment:
Why do we need sorted paragraphs?  Currently cannot think of a case where it is not sorted.

Reply:
@TakaValley Did you see any chance the paragraphs are not sorted when you updateed `sample_analyze_ready.py`?

Reply:
Remove the `sort` across samples as I don't see any difference without it.

Reply:
Update: we should add the `sort` based on this ADO task: https://msazure.visualstudio.com/Cognitive%20Services/_queries/edit/26625193/?triage=true

Reply:
Looks good to me